puddles/Backplane/src/ros.c

```diff
@@ -125,14 +127,27 @@ const float depth_variance = 0.003;
 rcl_publisher_t adc_publisher;
 riptide_msgs2__msg__ElectricalReadings status_msg = {0};
 
+// Electrical Commands
+rcl_subscription_t elec_command_subscriber;
+riptide_msgs2__msg__ElectricalCommand elec_command_msg;
+
 // ========================================
 // Executor Callbacks
 // ========================================
 
 static void dshot_subscription_callback(const void * msgin) {
     const riptide_msgs2__msg__DshotCommand * msg = (const riptide_msgs2__msg__DshotCommand *)msgin;
     dshot_update_thrusters(msg->values);
-    dshot_command_received = true;
+    // dshot_command_received = true;
```

Comment:
Re-enable dshot telemetry publishing?

Reply:
Thats what i was missing. Ill put it back in